### ISSUE TITLE
XDrip does not alarm for high and low glucose levels

### ISSUE THREAD

**Issue by Runner25** [NightscoutFoundation/xDrip#899]
Alarms are set and work properly after reboot of the smartphone. After a certain time (few hours) something causes that alarms are logged and listed in the status line but neither sound nor vibration is working on the phone. In parallel the vibration alarm on my smart watch is working well.
Actual XDrip version 14a03a7-2019.05.18.

**Comment by fromorbonia** [comment]
We've noticed something similar, and was about to report that we can't haven't been able to get the "missed signal" to sound an alarm for a while.
Running on an LG G6, although I've just looked at my version 2019.01.10 1901102241. Will update and report back.

**Comment by fromorbonia** [comment]
Tried a number of tests and it is definitely not triggering an audible alarm on the missed signal reading. Hard to test the other alerts as out MiaoMiao has broken :-(

**Comment by osman-brian** [comment]
I just came here to report something similar, might be related. I think that sometime in the last few versions, the audio portion of alerts stopped working unless the Media volume is turned up. I usually keep my phone (Pixel 3XL) with the Ring & Alarm volumes turned up, and Media volume turned off. I don't have the phone in Vibrate or Silent mode (so other app notifications and phone calls do play sounds). XDrip no longer does - it only vibrates. I've tested the various alert settings - setting "Override Silent Mode" fixes it, as does turning my media volume up to the minimum (1 tick above zero).

*However*, the volume of the alerts is now always VERY loud, even with all volumes (Alarm, Ring, Media) turned to their lowest (non-zero) setting.

**Comment by Runner25** [comment]
Brian, this seem to be another issue than mine. My alarms are working properly when just rebootet the phone. Sometimes they stop working after 2 hours, today they were still working after 24h. Now, after 26h, they do not work anymore. When I force to test them in the setup, the alarm gets properly logged but no signal appears. I can never be sure if alarms are working so they are useless. Don't think it's a matter of settings as they work after rebooting.

**Comment by Navid200** [comment]
@Runner25 xDrip has changed a lot since the time you opened this issue.  Are you still experiencing this problem today?
If not, would you please close the issue?
Thanks

**Comment by Navid200** [comment]
Closing due to inactivity
